packages/balancer-apr/package.json

```diff
@@ -37,6 +37,7 @@
   "dependencies": {
     "@vercel/postgres": "^0.5.1",
     "drizzle-orm": "^0.29.1",
+    "p-throttle": "^6.0.0",
```

Comment:
Why?

Reply:
I was getting rate limited by the contracts even with multicall configuration `batchSize` and `wait`. At this point prefered to over kill and get all the data in instead of investing more time debugging it